lib/engine/action/simple_bid.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'base'
```

Comment:
let's just use bid and not require a company or corporation

Reply:
Yeah, I see now I was being stupid. I conflated the values returned by actions with the actual Action classes.

Reply:
And now I see I wasn't being stupid. I need to distinguish between "bid" and "simple_bid" so that bids are placed in the correct location in the auction view. I guess this is an excellent reason to add a new type of auction view (that isn't an "Auction").

Reply:
Now using "auctioning" to distinguish between a bid on a company and a bid for turn order